src/utils/fetchClient.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Avoid using 'eslint-disable' unless absolutely necessary. If you still need to use it, consider disabling only specific rules instead of all rules using '@typescript-eslint/no-explicit-any'.

Reply:
done